samples/subsys/usb/mass/boards/rpi_pico.overlay

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2024 Kelly Lord
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/delete-node/ &code_partition;
+
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		/*
+		 * Usable flash. Starts at 0x100, after the bootloader. The partition
+		 * size is 1MB minus the 0x100 bytes taken by the bootloader.
```

Comment:
Is the "bootloader" wording correct here? Maybe you wanted to say "taken by the firmware" instead.

Reply:
Looks like the comment is copied and updated from the original `rpi_pico` device tree. I think the comment is still correct as the second stage bootloader does go from 0x0 to 0x100.

However, doesn't this file replace the `flash0` node completely? The current Pico device tree has a `second_stage_bootloader` node that represents that boot loader partition, which I think still needs to remain.